mage/query-modules/cpp/meta.md

```diff
@@ -0,0 +1,199 @@
+---
+id: meta
+title: meta
+sidebar_label: meta
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+  backgroundColor: color,
+  borderRadius: '2px',
+  color: '#fff',
+  padding: '0.2rem',
+}}>
+{children}
+</span>
+);
+
+
+The **meta** module provides a set of procedures for generating metadata about the database.
+
+[![docs-source](https://img.shields.io/badge/source-util_module-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/tree/main/cpp/meta_module)
+
+| Trait               | Value                                                 |
+| ------------------- | ----------------------------------------------------- |
+| **Module type**     | <Highlight color="#FB6E00">**algorithm**</Highlight>  |
+| **Implementation**  | <Highlight color="#FB6E00">**C++**</Highlight>        |
+| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight> |
+
+### Procedures
+
+## stats
+
+The stats procedure returns the following metadata about the graph:
+- `labelCount` ➡ number of unique labels in nodes
+- `relationshipTypeCount` ➡ number of unique relationship types (labels)
+- `nodeCount` ➡ number of nodes in the graph
+- `relationshipCount` ➡ number of relationships in the graph
+- `labels` ➡ map with the following (key, value) pairs:
+  - `label` : number_of_occurrences
+- `relationshipTypes` ➡ map with the following (key, value) pairs:
+  - `(:label)-[:relationship_type]->()` : number_of_occurrences
+  - `()-[:relationship_type]->(:label)` : number_of_occurrences
+  - `()-[:relationship_type]->()` : number_of_occurrences
+- `relationshipTypesCount` ➡ map with the following (key, value) pairs:
+  - `relationship_type` : number_of_occurrences
+- `stats` ➡ map which contains all of the above
+
+It is split into two version which return the same metadata:
+- stats_online - works in **O(1)** and requires setting up a trigger
+- stats_offline - traverses the whole graph
```

Comment:
```suggestion
- `stats_online` - works in **O(1)** and requires setting up a trigger
- `stats_offline` - traverses the whole graph
```

Reply:
Are you listing the same outputs three times then? 
I don't think i would do that... i would just have stats online and offline and then this knowledge

- stats_online - works in **O(1)** and requires setting up a trigger
- stats_offline - traverses the whole graph

can be in the intro, or each in its own section. 

It's confusing like this + different from all the other mage pages